package.json

```diff
@@ -1,4 +1,17 @@
 {
+  "name": "nextjs-blog-theme",
+  "description": "A customizable Next.js and Tailwind blog starter. Designed by the Bejamas agency.",
+  "author": "Bejamas and Netlify Inc.",
+  "contributors": [
+    "Charlie Gerard",
+    "Prince Wilson",
+    "Tara Manicsic",
+    "Thom Krupa"
+  ],
+  "homepage": "https://github.com/netlify-templates/nextjs-blog-theme",
+  "bugs": {
+    "url": "https://github.com/netlify-templates/nextjs-blog-theme/issues"
+  },
   "private": true,
```

Comment:
should this be `false` instead 🤔  I don't know if we really need it cause this repo isn't really a package we're publishing on npm

Reply:
Yeah I wasn't certain either. I will remove it!